include/SFML/Graphics/Font.hpp

```diff
@@ -322,12 +329,6 @@ class SFML_GRAPHICS_API Font
         std::vector<Row> rows;       //!< List containing the position of all the existing rows
     };
 
-    ////////////////////////////////////////////////////////////
-    /// \brief Free all the internal resources
-    ///
-    ////////////////////////////////////////////////////////////
-    void cleanup();
```

Comment:
If an API refactor lets us remove manual cleanup functions then we know we're doing something right.